class.phpmailer.php

```diff
@@ -1364,7 +1364,8 @@ public function postSend()
      */
     protected function sendmailSend($header, $body)
     {
-        if (!empty($this->Sender)) {
+        // CVE-2016-10033, CVE-2016-10045: Don't pass -f if characters will be escaped by escapeshellcmd when popen is called due to safe mode.
+        if (!empty($this->Sender) || ini_get('safe_mode')) {
```

Comment:
Yes, you're right. I originally rearranged the if statement a bit and didn't properly refactor.